code/Data exploration.R

```diff
@@ -53,8 +56,37 @@ data <- data %>% arrange(DATETIME);
 ##fill in missing temperature (down fill)
 data <- data %>% fill(TEMPERATURE)
 
-##convert DATETIME to date
-data$DATETIME = ymd_hms(data$DATETIME);
+
+##create table with period of initial forecast
+initial_forecast <- forecastdemand_nsw %>% 
+  filter(PERIODID==1) %>% 
+  select(DATETIME,
+         FORECASTDEMAND)%>%
+  rename('INITIAL_FORECAST'='FORECASTDEMAND')
+
+##create table with period of final forecast
+final_forecast_period <- forecastdemand_nsw %>% 
+  group_by(DATETIME) %>% 
+  summarise_at(vars(PERIODID),
+               list(PERIODID = max))
+final_forecast <- forecastdemand_nsw %>%
+  inner_join(final_forecast_period,by=c('DATETIME','PERIODID')) %>%
```

Comment:
[clarification]: we're inner joining here to filter to max period of each datetime?

Reply:
that's right, as in this instance I'm only after the final estimate of each period